src/base.js

```diff
@@ -1,95 +1,88 @@
+'use strict';
 
-const service = new SERVICE.Service();
-const htmlHelper = new HTML_HELPER.HtmlHelper();
-const ELEMENTS = {
+var service = new SERVICE.Service();
+var htmlHelper = new HTML_HELPER.HtmlHelper();
+var ELEMENTS = {
 	container: document.getElementById('sources'),
 	modalBody: document.getElementById('modalBody'),
 	search: document.getElementById('search'),
 	searchId: document.getElementById('searchId')
 };
-const COMMON_ERROR = 'An error occurred during getting information!';
-const NOTEXIST_ERROR = 'Current source does not exist!';
+var COMMON_ERROR = 'An error occurred during getting information!';
+var NOTEXIST_ERROR = 'Current source does not exist!';
 
 window.onload = () => {
 	BindAutocomplete(sources);
 	LoadMainPage(sources);
 	addEventListeners();
-}
-
-const addEventListeners = () => {
-
-	document.getElementById('searchBtn').addEventListener('click', function () {
-		LoadArticles(ELEMENTS.searchId.value);
-	});
+};
 
+var addEventListeners = () => {
 
-	document.getElementById('mainPage').addEventListener('click', function () {
-		LoadMainPage(sources);
-	});
-}
+	document.getElementById('searchBtn').addEventListener('click', () => { LoadArticles(ELEMENTS.searchId.value); });

```

Comment:
Read this `https://en.wikipedia.org/wiki/Line_wrap_and_word_wrap`, please.
```
document.getElementById('searchBtn')
  .addEventListener('click', () => { LoadArticles(ELEMENTS.searchId.value); });
```

Reply:
done